metapool/tests/test_prep.py

```diff
@@ -163,7 +164,7 @@ def _check_run_230207_M05314_0346_000000000_KVMGL(self, obs):
              1, 'Echo 550', 'ABTX_20230208_11052_Plate_238_11.8.21.RK.FH_A1',
              float('nan'), float('nan'), 1317793, float('nan'), 'V4', 'A1',
              'ABTX_20230208_ABTX_11052', '108379Z', 'sample.1', 'HT', 122822,
-             1331807, float('nan')],
+             1331807, float('nan'), 'Z1', float('nan')],
```

Comment:
`Z1` isn't a valid 96 well coordinate. Can the upstream code error politely if an invalid coordinate is specified?

Reply:
Thanks! Doing both would be defensive